### DISCUSSION TITLE
Type instantiation is excessively deep and possibly infinite

### DISCUSSION THREAD

**Discussion by KH93b** [konvajs/konva#1661] [General]
import React from 'react';
import { Spring, animated } from '@react-spring/konva';

export const Home = () => {
	return (
		<Spring
			from={{ x: 0 }}
			to={{
				x: 150,
			}}
		>
			{props => <animated.Rect />}
		</Spring>
	);
};

animated.Rect => Type instantiation is excessively deep and possibly infinite.

Why @react-spring/konva + react-konva + typescript = ERROR :)

**Comment by lavrton** [comment]
I can't do here anything from konva side.
Relevant: https://github.com/pmndrs/react-spring/issues/1515
